hw/ip/tlul/generic_dv/xbar_sim_cfg.hjson

```diff
@@ -0,0 +1,165 @@
+  sim_tops: ["-top {dut}_bind"]
+
+  // Default iterations for all tests - each test entry can override this.
+  reseed: 50
+
+  // Default UVM test and seq class name.
+  uvm_test: xbar_base_test
+  uvm_test_seq: xbar_base_vseq
+
+  // Add xbar specific exclusion files.
+  vcs_cov_excl_files: ["{proj_root}/hw/top_earlgrey/ip/{dut}/dv/autogen/cov/xbar_cov_excl.el"]
+
+  // List of test specifications.
+  tests: [
+    {
+      name: xbar_sanity
+      uvm_test_seq: xbar_sanity_vseq
+    }
+
+    {
+      name: xbar_sanity_zero_delays
+      uvm_test_seq: xbar_sanity_vseq
+      run_opts: ["+zero_delays=1"]
+    }
+
+    {
+      name: xbar_sanity_large_delays
+      uvm_test_seq: xbar_sanity_vseq
+      run_opts: ["+max_host_req_delay=1000",
+                 "+max_host_rsp_delay=1000",
+                 "+max_device_req_delay=1000",
+                 "+max_device_rsp_delay=1000"]
+    }
+
+    {
+      name: xbar_sanity_slow_rsp
+      uvm_test_seq: xbar_sanity_vseq
+      run_opts: ["+max_host_req_delay=10",
+                 "+max_host_rsp_delay=1000",
+                 "+max_device_req_delay=1000",
+                 "+max_device_rsp_delay=10"]
+    }
+
+    {
+      name: xbar_random
+      uvm_test_seq: xbar_random_vseq
+    }
+
+    {
+      name: xbar_random_zero_delays
+      uvm_test_seq: xbar_random_vseq
+      run_opts: ["+zero_delays=1"]
+    }
+
+    {
+      name: xbar_random_large_delays
+      uvm_test_seq: xbar_random_vseq
+      run_opts: ["+max_host_req_delay=1000",
+                 "+max_host_rsp_delay=1000",
+                 "+max_device_req_delay=1000",
+                 "+max_device_rsp_delay=1000"]
+    }
+
+    {
+      name: xbar_random_slow_rsp
+      uvm_test_seq: xbar_random_vseq
+      run_opts: ["+max_host_req_delay=10",
+                 "+max_host_rsp_delay=1000",
+                 "+max_device_req_delay=1000",
+                 "+max_device_rsp_delay=10"]
+    }
+
+    {
+      name: xbar_access_same_device
+      uvm_test_seq: xbar_access_same_device_vseq
+    }
+
+    {
+      name: xbar_access_same_device_slow_rsp
+      uvm_test_seq: xbar_access_same_device_vseq
+      run_opts: ["+max_host_req_delay=10",
+                 "+max_host_rsp_delay=1000",
+                 "+max_device_req_delay=1000",
+                 "+max_device_rsp_delay=10"]
+    }
+
+    {
+      name: xbar_same_source
+      uvm_test_seq: xbar_same_source_vseq
+    }
+
+    {
+      name: xbar_error_random
+      uvm_test: xbar_error_test
+      uvm_test_seq: xbar_random_vseq
+    }
+
+    {
+      name: xbar_unmapped_addr
+      uvm_test_seq: xbar_unmapped_addr_vseq
+    }
+
+    {
+      name: xbar_error_and_unmapped_addr
+      uvm_test: xbar_error_test
+      uvm_test_seq: xbar_unmapped_addr_vseq
+    }
+
+    {
+      name: xbar_stress_all
+      uvm_test_seq: xbar_stress_all_vseq
+    }
+
+    {
+      name: xbar_stress_all_with_rand_reset
+      uvm_test_seq: xbar_stress_all_with_rand_reset_vseq
+    }
```

Comment:
can't reuse the common stress_test.hjson, since xbar doesn't use ip_common_vseq to do xbar_stress_all w/ reset and  doesn't use cip_lib as no csr.